lnrpc/stateservice.ts

```diff
@@ -155,24 +152,20 @@ export const SubscribeStateResponse = {
           break;
       }
     }
-
     return message;
   },
 
   fromJSON(object: any): SubscribeStateResponse {
-    const message = createBaseSubscribeStateResponse();
-    message.state =
-      object.state !== undefined && object.state !== null
-        ? walletStateFromJSON(object.state)
-        : 0;
-    return message;
+    return {
+      state: isSet(object.state) ? walletStateFromJSON(object.state) : 0,
+    };
   },
 
   toJSON(message: SubscribeStateResponse): unknown {
-    const object: any = {};
+    const obj: any = {};
     message.state !== undefined &&
```

Comment:
TBH I really hate this syntax- I prefer a bracket-less if statement for stuff like this.

Reply:
What about this?

```
return {
  state: message.state ? walletStateToJSON(message.state) : undefined
}
```